tests/test_utils.py

```diff
@@ -219,15 +219,16 @@ def test_str_to_unicode():
     s = ["test1", "test2", b"test3"]
     exp_answer = ["test1", "test2", "test3"]
     assert_array_equal(utils.str_to_unicode(s), exp_answer)
-
     # set of str
     s = {"test1", "test2", b"test3"}
     exp_answer = {"test1", "test2", "test3"}
+    # Fix: assert_equal not working
```

Comment:
Not sure how to interpret this comment - do you plan to fix it?  If so, we can mark this PR as a draft or work in progres...

Reply:
Yes, I will try. I have found **assert_equal** only works in the string object. In this case, the values are in the dictionary and tuple format.

Reply:
I have fixed the issue here [3e57c2a](https://github.com/pyne/pyne/pull/1472/commits/3e57c2ae4c0dee7e122067e6c3b836cb71f87cbb)